hawc/apps/assessment/api/viewsets.py

```diff
@@ -271,6 +271,15 @@ def public(self, request):
         serializer = serializers.AssessmentSerializer(queryset, many=True)
         return Response(serializer.data)
 
+    @action(detail=False, permission_classes=(permissions.IsAuthenticated,))
+    def team_member(self, request):
+        """Get the list of assessments where the user is a team member or project manager."""
+        queryset = self.model.objects.filter(
+            Q(project_manager=request.user) | Q(team_members=request.user)
+        )
+        serializer = serializers.AssessmentSerializer(queryset, many=True)
```

Comment:
This serializer or the AssessmentMiniSerializer?

Reply:
this one is good I think, thanks!